src/components/Icon/Expensicons.ts

```diff
@@ -386,4 +387,5 @@ export {
     Filters,
     CalendarSolid,
     Filter,
+    Table,
```

Comment:
You should do the same with `SpreadsheetComputer` icon, right?

Reply:
✅